INSTALL.md

```diff
@@ -0,0 +1,36 @@
+# How to install the workflow from scratch on a Linux machine
+
+## Install Lmod
+
+To control which version is used for each tools and dependencies for the workflow, a module environnement system as lmod is used.
+
+Follow instructions for installing it on [lmod.readthedocs.io](http://lmod.readthedocs.io/en/latest/index.html)
+
+## Install slurm
+
+The idea here is to emulate a slurm ordonnancing system on a non-cluster machine to run the workflow.
+
+Enable
+OPTIONS="--force --key-file /etc/munge/munged.key --num-threads 1"
+on '/etc/default/munge'
+
+## Tools and dependencies
+https://sourceforge.net/projects/bio-bwa/files/
+- bwa 0.7.8
+https://software.broadinstitute.org/gatk/download/index
+- GATK 3.3-0
+- GATK 3.6 [MuTect2]
+https://sites.google.com/site/strelkasomaticvariantcaller/home/download
+- Strelka 1.0.15
+- gcc 4.9.2
+- java sun_jdk 1.8.0_92
```

Comment:
We should stick to java sun_jdk 1.8.0_40 only for now, as irma has only this version of 1.8